src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,95 @@
+package duke.command;
+
+import java.time.DateTimeException;
+
+import duke.main.Storage;
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todos;
+
+/**
+ * A class for the command for adding new task to list based on user input.
+ */
+public class AddCommand extends Command {
```

Comment:
Class names should be nouns, if I'm not wrong. This would apply to the ByeCommand, DeleteCommand etc classes as well. Not too sure if this is applicable in your case though.